API `items` endpoint broken

```
[ERROR] TypeError: Object of type generator is not JSON serializable
Traceback (most recent call last):
  File "/var/task/lambda_function.py", line 167, in lambda_handler
    return response(transform(item) for item in items)
  File "/var/task/lambda_function.py", line 29, in response
    "body": json.dumps(body)
  File "/var/lang/lib/python3.7/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/var/lang/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/var/lang/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/var/lang/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
```

We should get a release with a fix for this ASAP. We also need a way to define tests for the API in the mid-term.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+
+## [v0.5.3] - 2022-02-10
+
+### Fixed
+
+- API `items` endpoint fixed ([#114])
+
+
 ## [v0.5.2] - 2022-02-09
 
 ### Fixed
@@ -278,7 +286,8 @@ Initial release
 
 
 
-[Unreleased]: https://github.com/cirrus-geo/cirrus-geo/compare/v0.5.2...main
+[Unreleased]: https://github.com/cirrus-geo/cirrus-geo/compare/v0.5.3...main
+[v0.5.3]: https://github.com/cirrus-geo/cirrus-geo/compare/v0.5.2...v0.5.3
 [v0.5.2]: https://github.com/cirrus-geo/cirrus-geo/compare/v0.5.1...v0.5.2
 [v0.5.1]: https://github.com/cirrus-geo/cirrus-geo/compare/v0.5.0...v0.5.1
 [v0.5.0]: https://github.com/cirrus-geo/cirrus-geo/compare/v0.5.0a5...v0.5.0
@@ -313,6 +322,7 @@ Initial release
 [#107]: https://github.com/cirrus-geo/cirrus-geo/issues/107
 [#108]: https://github.com/cirrus-geo/cirrus-geo/issues/108
 [#111]: https://github.com/cirrus-geo/cirrus-geo/issues/111
+[#114]: https://github.com/cirrus-geo/cirrus-geo/issues/114
 
 [#71]: https://github.com/cirrus-geo/cirrus-geo/pull/72
 [#72]: https://github.com/cirrus-geo/cirrus-geo/pull/72

--- a/src/cirrus/builtins/functions/api/lambda_function.py
+++ b/src/cirrus/builtins/functions/api/lambda_function.py
@@ -164,7 +164,7 @@ def lambda_handler(event, context):
             sort_ascending=sort_ascending,
             sort_index=sort_index,
         )
-        return response(transform(item) for item in items)
+        return response({'items': [transform(item) for item in items['items']]})
     else:
         # get individual item
         item = statedb.dbitem_to_item(statedb.get_dbitem(payload_id))

--- a/tests/fixture_data/build/hashes.json
+++ b/tests/fixture_data/build/hashes.json
@@ -100,8 +100,8 @@
         "size": 79
     },
     "lambdas/api/lambda_function.py": {
-        "shasum": "429d7270427916b47daef95ae508f132d4f48a1e0820527e93b6f095951c8abb",
-        "size": 5557
+        "shasum": "7654e96d5eb254f32da2ba48e071ecaca64253ef87e7126b37942b97dd6ac2bb",
+        "size": 5579
     },
     "lambdas/feed-rerun/requirements.txt": {
         "shasum": "76941e32ac85dc575cd3ed7a73ac60e271a6d284634aba35cf7740bc009792ae",

